src/lib/bootloader/udev_mapping.rb

```diff
@@ -79,7 +79,7 @@ def udev_to_kernel(dev)
       # in mode config if not found, then return itself
       return dev if Yast::Mode.config
 
-      devices = Y2Storage::BlkDevice.all(staging)
+      devices = staging.disk_devices
```

Comment:
Tests failed. Main problem is that disk_devices gets only disk and similar devices, but we need all devices including also partitions, that is why tests failed, as it tries to get kernel name for partition.

Reply:
in general we need something like `staging.find_by_udev_name` or at least something like `staging.devices_with_udev_links`